src/odbc/odbc.c

```diff
@@ -1329,8 +1329,8 @@ SQLSetEnvAttr(SQLHENV henv, SQLINTEGER Attribute, SQLPOINTER Value, SQLINTEGER S
 		break;
 	case SQL_ATTR_OUTPUT_NTS:
 		/* TODO - Make this really work */
-		/* env->attr.output_nts = i_val; */
-		env->attr.output_nts = SQL_TRUE;
+		env->attr.output_nts = i_val;
```

Comment:
I cannot see any tests that prove that is safe to do it.

Reply:
OK, yeah, this change makes no difference to your tests, and we don't otherwise use the ODBC library these days (having settled on the CT-LIB API).  I'll drop it.